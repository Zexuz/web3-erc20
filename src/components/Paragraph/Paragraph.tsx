interface ParagraphProps {
  text: string;
}

export const Paragraph = ({ text }: ParagraphProps) => {
  return <p className={"font-poppins text-sm font-medium"}>{text}</p>;
};
