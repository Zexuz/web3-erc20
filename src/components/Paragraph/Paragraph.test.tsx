import renderer from "react-test-renderer";
import { Paragraph } from "./Paragraph.tsx";

describe("Paragraph", () => {
  it("should render the text", () => {
    const tree = renderer.create(<Paragraph text="test" />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
