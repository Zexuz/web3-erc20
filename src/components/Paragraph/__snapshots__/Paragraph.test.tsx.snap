// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph should render the text 1`] = `
<p
  className="font-poppins text-sm font-medium"
>
  test
</p>
`;
